content/en/docs/ambient/install/helm-installation/index.md

```diff
@@ -114,8 +138,48 @@ After installing ambient mode with Helm, you can follow the [Deploy the sample a
 
 ## Uninstall
 
-You can uninstall Istio and its components by uninstalling the charts
-installed above.
+If you installed with the `ambient` wrapper chart above, you can uninstall Istio and its components by uninstalling that chart:
```

Comment:
```suggestion
If you installed with the simple `ambient` chart above, you can uninstall Istio and its components by uninstalling that chart:
```

maybe don't put this before the gateway instructions?

Reply:
Yeah the idea was to exactly reverse the install directions but I dorked the wording here.